test/components/evaluators/test_document_ndcg.py

```diff
@@ -0,0 +1,147 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+
+from haystack import Document
+from haystack.components.evaluators.document_ndcg import DocumentNDCGEvaluator
+
+
+def test_run_with_scores():
```

Comment:
Really good tests overall. Maybe if wanted, another test can be added for `run` with multiple lists of documents?

Reply:
Good point. I added one test to address this.